core/src/main/java/tc/oc/pgm/shops/ShopKeeper.java

```diff
@@ -0,0 +1,65 @@
+package tc.oc.pgm.shops;
+
+import static tc.oc.pgm.util.bukkit.BukkitUtils.colorize;
+
+import com.google.common.collect.ImmutableList;
+import javax.annotation.Nullable;
+import org.bukkit.ChatColor;
+import org.bukkit.Location;
+import org.bukkit.World;
+import org.bukkit.entity.Entity;
+import org.bukkit.metadata.FixedMetadataValue;
+import tc.oc.pgm.api.PGM;
+import tc.oc.pgm.api.match.Match;
+import tc.oc.pgm.points.PointProvider;
+import tc.oc.pgm.util.nms.NMSHacks;
+
+public class ShopKeeper {
+
+  public static final String METADATA_KEY = "SHOP_KEEPER";
+
+  private final String name;
+  private final ImmutableList<PointProvider> location;
+  private final Class<? extends Entity> type;
+  private final Shop shop;
+
+  public ShopKeeper(
+      @Nullable String name,
+      ImmutableList<PointProvider> location,
+      Class<? extends Entity> type,
+      Shop shop) {
+    this.name = name;
+    this.location = location;
+    this.type = type;
+    this.shop = shop;
+  }
+
+  @Nullable
+  public Location getLocation(Match match) {
+    // TODO: support multiple shopkeeper locations?
```

Comment:
I would think we should support this? Let's say I'm implementing this for Rift; I'd have 4 shop keepers, each would be on both sides.

Reply:
That is simply 2 shop keepers, for the same shop, i don't think it's relevant

Reply:
See my commit from the other day, I looked into multiple shopkeeper locations before I saw your comment and thought it was pretty useful. 

For example you can do something like this:
```xml
    <shopkeeper name="`5Purple `7Item Shop" mob="Villager" shop="items">
      <point yaw="-99" pitch="-22">-88.5,14,-103.5</point>
      <point yaw="90" pitch="-22">100.5,16,-103.5</point>
      <point yaw="180" pitch="-22">10,18,-10.5</point>
      <point yaw="0" pitch="-22">105,20,200.5</point>
    </shopkeeper>
```
and it'll spawn the same shop keeper at all those locations. While smaller maps may not find it useful, might be nice for larger scale ones.